tests/test_include_markdown.py

```diff
@@ -133,3 +134,89 @@ def test_include_markdown_filepath_error(page):
 
         with pytest.raises(ValueError):
             _on_page_markdown(page_content, page(f_includer.name))
+
+
+@pytest.mark.parametrize('should_rewrite', [True, False, None])
+def test_include_markdown_relative_rewrite(page, tmp_path, should_rewrite):
+    option_value = {
+        True: 'rewrite_relative_urls=true',
+        False: 'rewrite_relative_urls=false',
+        None: '',
+    }[should_rewrite]
+
+    includer_path = tmp_path / 'includer.md'
+    includer_path.write_text(textwrap.dedent(f'''
+        # Heading
+
+        {{%
+            include-markdown "docs/page.md"
+            start="<!--start-here-->"
+            end="<!--end-here-->"
+            {option_value}
+        %}}
+    '''))
+
+    (tmp_path / 'docs').mkdir()
+    included_file_path = tmp_path / 'docs' / 'page.md'
+    included_file_path.write_text(textwrap.dedent('''
+        # Subpage Heading
+        <!--start-here-->
+        Here's [a link](page2.md) and here's an image: ![](image.png)
+
+        Here's a [reference link][ref-link].
+
+        [ref-link]: page3.md
+        <!--end-here-->
+    '''))
+
+    output = _on_page_markdown(
+        includer_path.read_text(),
+        page(str(includer_path))
+    )
+
+    if should_rewrite is True or should_rewrite is None:
```

Comment:
I think that reference equality is not needed here, so what about this?

```suggestion
    if should_rewrite in [True, None]:
```